assets/schema/ensemble_schema.json

```diff
@@ -1712,30 +1712,39 @@
                   "type": "string",
                   "description": "Field name that your server is expecting (default files)"
                 },
-                "options": {
-                  "type": "object",
-                  "properties": {
-                    "allowMultiple": {
-                      "type": "boolean",
-                      "description" : "Allow users to pick multiple files (default False)"
-                    },
-                    "allowCompression": {
-                      "type": "boolean",
-                      "description" : "It will allow media to apply the default OS compression (default True)"
-                    },
-                    "allowedExtensions": {
-                      "type": "array",
-                      "description": "Allow files with specific extension e.g jpg, png, pdf"
-                    },
-                    "maxFileSize" : {
-                      "type": "integer",
-                      "description": "File size that is allowed (default 100 mb), If multiple is allow then sum of all files"
-                    },
-                    "overMaxFileSizeMessage" : {
-                      "type": "string",
-                      "description": "Error message to show when selected files size is above maxFileSize."
-                    }
-                  }
+                "files": {
+                  "type": "string",
+                  "description": "Binded files from sources like camera or picker. e.g ${filePicker.files}, ${camerId.mediaList}"
+                }
+              }
+            }
+          }
+        },
+        {
+          "properties": {
+            "pickFiles": {
```

Comment:
this needs an id too?

Reply:
Yes, id is required because all files that are picked will be bind to that id.

```yaml
Button:
  label: Pick Files
  onTap:
    pickFiles:
      id: picker
```

Here all picked files can be accessed by `${picker.files}`